src/paxos.rs

```diff
@@ -165,7 +165,7 @@ where
     }
 
     /// Returns the decided entries since the last call of this function.
-    pub fn get_decided_entries(&mut self) -> &[Entry<R>] {
+    pub fn get_last_decided_entries(&mut self) -> &[Entry<R>] {
```

Comment:
I think "get_last" can sound like the last entry only. How about `get_latest_decided_entries`?